conf/readme.yaml

```diff
@@ -0,0 +1,23 @@
+data:
```

Comment:
Did you intend to include all these .yaml configuration files? I feel like the one we have for defaults is already enough, but maybe there are some use cases I haven't thought of.

Reply:
These were unintended, Thank you. I will delete the files.